File: ai/go/detect-gemini.go
// ruleid: detect-gemini
import "github.com/google/generative-ai-go/genai"
import "google.golang.org/api/option"

ctx := context.Background()
// Access your API key as an environment variable (see "Set up your API key" above)
// ruleid: detect-gemini
client, err := genai.NewClient(ctx, option.WithAPIKey(os.Getenv("API_KEY")))
if err != nil {
    log.Fatal(err)
}
defer client.Close()

model := client.GenerativeModel("gemini-1.5-flash")

File: ai/typescript/detect-gemini.ts
// ruleid: detect-gemini
const { GoogleGenerativeAI } = require("@google/generative-ai");

// Access your API key as an environment variable (see "Set up your API key" above)
// ruleid: detect-gemini
const genAI = new GoogleGenerativeAI(process.env.API_KEY);

// ruleid: detect-gemini
const model = genAI.getGenerativeModel({ model: "gemini-1.5-flash"});